test/e2e/framework/configmanager/constants.go

```diff
@@ -54,5 +54,13 @@ const (
 	DefaultYuniKornScheme = "http"
 
 	DefaultYuniKornConfigMap = "yunikorn-configs"
+	DefaultPluginConfigMap   = "yunikorn-configs"
 	DefaultPolicyGroup       = "queues.yaml"
 )
+
+func GetConfigMapName() string {
+	if YuniKornTestConfig.Plugin {
+		return DefaultPluginConfigMap
+	}
+	return DefaultYuniKornConfigMap
+}
```

Comment:
The config names do not differ between plugin and default mode.
Code overhead is not needed.